official/enfuse.lua

```diff
@@ -16,26 +17,93 @@
     along with darktable.  If not, see <http://www.gnu.org/licenses/>.
 ]]
 --[[
-RUN ENFUSE ON THE SELECTED IMAGES
-This script uses enfuse to merge the selected images into one HDR and imports the result.
-It only works on ldr images (like, JPEG).
-
-USAGE
-* require this script from your main lua file
-* it creates a new lighttable module
-
-TODO
-* remember the exposure_mu value in config when the slider is moved
-* make the output filename unique so you can use it more than once per filmroll
-* find a less stupid way to make sure the float value of exposure_mu gets turned into a string
-  with a decimal point instead of a comma in some languages
-* export images that are not ldr and remove them afterwards
+    enfuse - export the selected images and run enfuse in one of two modes
+    * Enfuse HDR - combine the exported images into a single HDR image
+    * Enfuse Focus Stack - Combine a set of focus stack images into a single image
+
+    USAGE
+    * require this script from your main lua file
+    * two new storage options are created, Enfuse HDR and Enfuse Focus Stack
+    * Enfuse HDR
+      * select the bracketed images to use for the hdr
+      * adjust exposure mu to change the brightness of the output image
```

Comment:
I guess I should have written it exposure-mu.  It is an option to enfuse that determines the center of the Gaussian exposure weight curve.